src/GlobalSearch/DateOfBirth.test.js

```diff
@@ -1,37 +1,38 @@
 import React from 'react'
-import { shallow, mount } from 'enzyme'
+import { mount } from 'enzyme'
 import moment from 'moment'
+import { JSDOM } from 'jsdom'
 import DateOfBirth from './DateOfBirth'
 
+const jsdom = new JSDOM('<!doctype html><html lang="en"><body></body></html>')
+const { window } = jsdom
+global.window = window
+global.document = window.document
+
 describe('DateOfBirth', () => {
   it('Should render', () => {
-    shallow(<DateOfBirth />)
+    mount(<DateOfBirth />)
   })
 
   it('handleDateOfBirthChange is called during initialisation', () => {
     const handleDateOfBirthChange = jest.fn()
-    shallow(<DateOfBirth handleDateOfBirthChange={handleDateOfBirthChange} />)
+    mount(<DateOfBirth handleDateOfBirthChange={handleDateOfBirthChange} />)
     expect(handleDateOfBirthChange).toHaveBeenCalledWith({ valid: false, blank: true })
   })
 
   it('Entering parts of a date calls handleDateOfBirthChange with new outputs', () => {
     const handleDateOfBirthChange = jest.fn()
     const dob = mount(<DateOfBirth handleDateOfBirthChange={handleDateOfBirthChange} />)
     expect(handleDateOfBirthChange).toHaveBeenCalledWith({ valid: false, blank: true })
-    dob
-      .find('input[type="number"]')
-      .at(0)
-      .simulate('change', { target: { value: '1' } })
+    const textinputs = dob.find('input[type="number"]')
+
+    textinputs.at(0).simulate('change', { target: { value: '1' } })
     expect(handleDateOfBirthChange).toHaveBeenCalledWith({ valid: false, blank: false })
-    dob
-      .find('input[type="number"]')
-      .at(1)
-      .simulate('change', { target: { value: '1' } })
+    textinputs.at(0).instance().value = '1'
```

Comment:
`simulate` doesn't actually change the input value, hence have to amend the value too